source/MaterialXGraphEditor/Graph.cpp

```diff
@@ -104,6 +104,13 @@ std::string getUserNodeDefName(const std::string& val)
     return result;
 }
 
+static void EnableSRGBCallback(const ImDrawList*, const ImDrawCmd*)   {
```

Comment:
This is a minor note, but we use Allman-style braces in MaterialX, where the opening and closing brackets are placed on their own line, as in:

```
static void EnableSRGBCallback(const ImDrawList*, const ImDrawCmd*)
{
    glEnable(GL_FRAMEBUFFER_SRGB);
}
```